### PR TITLE
Update Post-Processor Docs

### PR DESCRIPTION
**Pull request by grubernaut**
Update Post-Processor docs to clarify sequences in packer templates.

### CODE REVIEW

File: website/source/docs/templates/post-processors.html.markdown [hunk 1]
```diff
@@ -96,6 +98,27 @@ compressed then uploaded, but the compressed result is not kept.
...
+{
+  "post-processors": [
+    [
+      {
+        "type": "vagrant",
+        "keep_input_artifact": false
+      },
+      {
+        "type": "atlas",
+        ...
```
Comment (cbednarski) [inline website/source/docs/templates/post-processors.html.markdown]: I think this is super useful. Can you expand this to include a full-blown Atlas example?

Comment (cbednarski) [inline website/source/docs/templates/post-processors.html.markdown]: We could also link to this page: https://packer.io/docs/post-processors/atlas.html

Comment (grubernaut) [inline website/source/docs/templates/post-processors.html.markdown]: Absolutely! Will add

File: website/source/docs/templates/post-processors.html.markdown [hunk 2]
```diff
@@ -82,6 +82,8 @@ sequence definition. Sequence definitions are used to chain together multiple
 post-processors. An example is shown below, where the artifact of a build is
 compressed then uploaded, but the compressed result is not kept.
 
+It is very important that any post processors that need to be ran in order, be sequenced!
```
Comment (pearkes) [inline website/source/docs/templates/post-processors.html.markdown]: "run in order"

Comment (pearkes) [inline website/source/docs/templates/post-processors.html.markdown]: (I think?)

Comment (justincampbell) [inline website/source/docs/templates/post-processors.html.markdown]: Yup

**Pull request merged**